src/SLCore.IntegrationTests/SLCoreTestRunner.cs

```diff
@@ -94,7 +96,7 @@ public async Task Start()
 
             slCoreHandle = new SLCoreHandle(new SLCoreRpcFactory(slCoreTestProcessFactory, slCoreLocator,
                     new SLCoreJsonRpcFactory(new RpcMethodNameTransformer()),
-                    new RpcDebugger(new FileSystem(), new DateTime(0)), //we pass datetime so we do not create extra file for each run
+                    new RpcDebugger($"{testName}.log"),
```

Comment:
This will put integration test runs in the same folder as the vsix logs, and that's inconvenient (folder switching) and confusing

Reply:
wdym

Reply:
Before, the logs were saved at `bin/.../slcore/<test_name>` folder along with sloop private files